CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-178'>AUTH-178</a>] -         Empty Realm in Identity Service
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-181'>AUTH-181</a>] -         Enable per-namespace host names in Bamboo-driven SAML tests
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-182'>AUTH-182</a>] -         Install jq binary in Bamboo elastic agents
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-183'>AUTH-183</a>] -         Keycloak stateful sets are left over after &quot;helm delete&quot;
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-189'>AUTH-189</a>] -         Keycloak PostgreSQL startup error when using helm upgrade
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-198'>AUTH-198</a>] -         Error message overflows login box
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-207'>AUTH-207</a>] -         APS http redirect_uri parameter for https-only requests
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-211'>AUTH-211</a>] -         Error pages don&#39;t render correctly
+</li>
+</ul>
+
+<h2>        Task
+</h2>
+<ul>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-15'>AUTH-15</a>] -         Move the activity-sso-idm project from the Activiti org to the Alfresco org
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-17'>AUTH-17</a>] -         A proof of concept that shows how to consume an identity token across several components of the platforms
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-25'>AUTH-25</a>] -         Design user token that will be used to represent the logged in user
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-26'>AUTH-26</a>] -         A proof of concept that shows how to consume an identity token across several components of the platforms
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-41'>AUTH-41</a>] -         A proof of concept that shows how to consume an identity token across several components of the platforms
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-46'>AUTH-46</a>] -         Investigate how AWS Cognito could be used directly
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-64'>AUTH-64</a>] -         Integrate code into ACS to implement the use of the internal auth token
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-67'>AUTH-67</a>] -         Ensure Keycloak Realm is Configurable at Deploy Time
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-68'>AUTH-68</a>] -         Use the Official Keycloak Helm Chart
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-69'>AUTH-69</a>] -         Update Hello World to demonstrate SSO
```

Comment:
Hello World app, not part of Identity Service

Reply:
I removed AUTH-69